fms/models/hf/config_utils/__init__.py

```diff
@@ -0,0 +1,22 @@
+from functools import partial
```

Comment:
Could we add a top-level docstring explaining how HF → FMS config mapping works via the registry?
This is now a central extension point, so documenting “how to register a new model” would be very helpful.

Reply:
Definitely! Added an in depth explanation for how to map the params here, and can also pull it out into more findable docs if we end up wanting steps for how to implement and test new models in general later on 😄